INSTALL

```diff
@@ -84,8 +84,8 @@ valgrind-xephyr.sh from your existing X session:
     mkdir -p ~/.config/herbstluftwm
     cp share/autostart ~/.config/herbstluftwm/
 
-5. add the share/herbstclient-completion to your
-   /usr/share/bash-completion/completions folder or source it in your bashrc
+5. add the share/herbstclient to your /usr/share/bash-completion/completions
```

Comment:
Isn't it `herbstclient-completion.bash` now? Maybe `copy` is better than `add` by the way

Reply:
Right, I forgot it's still in the pre-build dir structure. Thanks.